modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mongodb-sink-connector.adoc

```diff
@@ -112,6 +112,10 @@ MongoDB database:
 show collections
 ----
 
+== Using the Connectors API
+When using Connectors API, instead of `connection.url`, `connection.username` and `connection.password` configuration fields
+you can specify a value for the `connection.uri` field, of the form `mongodb+srv://username:password@cluster0.xxx.mongodb.net`.
```

Comment:
```suggestion
== Use the Connectors API
When using the Connectors API, instead of specifying a value for `connection.url`, `connection.username` and `connection.password`, 
you can specify a value for `connection.uri`, of the form `mongodb+srv://username:password@cluster0.xxx.mongodb.net`.
```